src/corehaptics.cs

```diff
@@ -0,0 +1,398 @@
+
+		[Abstract]
+		[Export ("maxValue")]
+		float MaxValue { get; }
+
+		[Abstract]
+		[Export ("defaultValue")]
+		float DefaultValue { get; }
+	}
+
+	interface ICHHapticDeviceCapability { } 
+
+	[iOS (13,0)][NoMac]
+	[Protocol]
+	interface CHHapticDeviceCapability {
+		[Abstract]
+		[Export ("supportsHaptics")]
+		bool SupportsHaptics { get; }
+
+		[Abstract]
+		[Export ("supportsAudio")]
+		bool SupportsAudio { get; }
+
+		// Protocols do not like BindAs yet we know is the enum CHHapticEventType
+		[Abstract]
+		[Export ("attributesForEventParameter:eventType:error:")]
+		[return: NullAllowed]
+		ICHHapticParameterAttributes GetAttributes (NSString eventParameter, string type, [NullAllowed] out NSError outError);
+
+		// Protocols do not like BindAs yet we know is the enum CHHapticEventType
+		[Abstract]
+		[Export ("attributesForDynamicParameter:error:")]
+		[return: NullAllowed]
+		ICHHapticParameterAttributes GetAttributes (NSString eventParameter, [NullAllowed] out NSError outError);
+	}
+
+	interface ICHHapticPatternPlayer { }
+
+	[Mac (10,15), iOS (13,0)]
+	[Protocol]
+	interface CHHapticPatternPlayer {
+		[Abstract]
+		[Export ("startAtTime:error:")]
+		bool Start (double time, [NullAllowed] out NSError outError);
+
+		[Abstract]
+		[Export ("stopAtTime:error:")]
+		bool Stop (double time, [NullAllowed] out NSError outError);
+
+		[Abstract]
+		[Export ("sendParameters:atTime:error:")]
+		bool Send (CHHapticDynamicParameter[] parameters, double time, [NullAllowed] out NSError outError);
+
+		[Abstract]
+		[Export ("scheduleParameterCurve:atTime:error:")]
+		bool Schedule (CHHapticParameterCurve parameterCurve, double time, [NullAllowed] out NSError outError);
+
+		[Abstract]
+		[Export ("cancelAndReturnError:")]
+		bool Cancel ([NullAllowed] out NSError outError);
+
+		[Abstract]
+		[Export ("isMuted")]
+		bool IsMuted { get; set; }
+	}
+
+	interface ICHHapticAdvancedPatternPlayer {}
+
+	[Mac (10,15), iOS (13,0)]
+	[Protocol]
+	interface CHHapticAdvancedPatternPlayer : CHHapticPatternPlayer {
+		[Abstract]
+		[Export ("pauseAtTime:error:")]
+		bool Pause (double time, [NullAllowed] out NSError outError);
+
+		[Abstract]
+		[Export ("resumeAtTime:error:")]
+		bool Resume (double time, [NullAllowed] out NSError outError);
+
+		[Abstract]
+		[Export ("seekToOffset:error:")]
+		bool Seek (double offsetTime, [NullAllowed] out NSError outError);
+
+		[Abstract]
+		[Export ("loopEnabled")]
+		bool LoopEnabled { get; set; }
+
+		[Abstract]
+		[Export ("loopEnd")]
+		double LoopEnd { get; set; }
+
+		[Abstract]
+		[Export ("playbackRate")]
+		float PlaybackRate { get; set; }
+
+		[Abstract]
+		[Export ("completionHandler", ArgumentSemantic.Assign)]
+		Action<NSError> CompletionHandler { get; set; }
+	}
+
+	[Mac (10,15), iOS (13,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface CHHapticEngine
+	{
+		[Static]
+		[Export ("capabilitiesForHardware")]
+		ICHHapticDeviceCapability GetHardwareCapabilities ();
```

Comment:
it's indeed a method
```objc
/*! @method capabilitiesForHardware
 	@abstract
 		Get the protocol that describes haptic and audio capabilities on this device.
 	@discussion
 		Detailed description on the capability protocol is in CHHapticDeviceCapability.h.
 */
+ (id<CHHapticDeviceCapability>)capabilitiesForHardware;
```
https://developer.apple.com/documentation/corehaptics/chhapticengine/3081788-capabilitiesforhardware?language=objc